entrypoint/site-config.sh

```diff
@@ -162,6 +165,7 @@ fi
 if [ "$SERVE_FILES" = "yes" ] ; then
         replace_in_file "${NGINX_PREFIX}server.conf" "%SERVE_FILES%" "include ${NGINX_PREFIX}serve-files.conf;"
         replace_in_file "${NGINX_PREFIX}serve-files.conf" "%ROOT_FOLDER%" "$ROOT_FOLDER"
+        replace_in_file "${NGINX_PREFIX}serve-files.conf" "%ROOT_SITE_SUBFOLDER%" "$ROOT_SITE_SUBFOLDER"
```

Comment:
This is not needed because we already edit ROOT_FOLDER.

Reply:
@bunkerity Thanks for the new release and all the great improvements. I'm trying to catch up and see what all's changed. 

I noticed that subfolders no longer work since these `site-config.sh` replacements were dropped in favor of the `serve-files.conf` template. I figured I'd point it out in case it took me a while to learn how this should be handled now. It's probably an easy fix.